lib/kuiq/model/job_manager.rb

```diff
@@ -64,6 +65,12 @@ def queues
         Sidekiq::Queue.all.map { |q| Kuiq::Model::Queue.new(q) }
       end
 
+      def metrics
+        q = Sidekiq::Metrics::Query.new
```

Comment:
It would be better to spell out the variable as `query` as otherwise, from a quick glance, I'd think `q` stands for `queue`.